src/scicookie/{{cookiecutter.project_slug}}/coc/NUMFOCUS_adapted_coc.md

```diff
@@ -0,0 +1,231 @@
+- Continuing to initiate interaction (such as photography, recording, messaging, or conversation) with someone after being asked to stop
+- Sustained disruption of talks, events, or communications, such as heckling of a speaker
+- Publishing (or threatening to post) other people's personally identifying information ("doxing"), such as physical or electronic addresses, without explicit permission
+- Other unethical or unprofessional conduct
+- Advocating for, or encouraging, any of the above behaviors
+
+### Reporting Guidelines
+
+If you believe someone is violating the code of conduct, please report this in a timely manner. Code of conduct violations reduce the value of the community for everyone. The {{ cookiecutter.project_name }} leadership team takes reports of misconduct very seriously and is committed to preserving and maintaining the welcoming nature of our community.
+
+**All reports will be kept confidential.**
+
+In some cases we may determine that a public statement will need to be made. If that's the case, the identities of all involved parties and reporters will remain confidential unless those individuals instruct us otherwise.
+
+All complaints will be reviewed and investigated and will result in a response that is deemed necessary and appropriate to the circumstances. The {{ cookiecutter.project_name }} team commits to maintaining confidentiality with regard to the reporter of an incident.
+
+For possibly unintentional breaches of the code of conduct, you may want to respond to the person and point out this code of conduct (either in public or in private, whatever is most appropriate). If you would prefer not to do that, please report the issue to {{ cookiecutter.project_name }} directly, or ask << PROJECT CoC Point Person(s) >> for advice in confidence (<< PROJECT CoC Point Person(s) Email address or other means of contact >>). Complete contact information is below, under "How to Submit a Report."
+
+Take care of each other. Alert {{ cookiecutter.project_name }} if you notice a dangerous situation, someone in distress, or violations of this code of conduct, even if they seem inconsequential.
+
+#### How to Submit a Report
+
+**If you feel your safety is in jeopardy or the situation is an emergency, we urge you to contact local law enforcement before making a report to {{ cookiecutter.project_name }}.** (In the U.S., dial 911.)
+
+{{ cookiecutter.project_name }} is committed to promptly addressing any reported issues. If you have experienced or witnessed behavior that violates the {{ cookiecutter.project_name }} Code of Conduct, please complete the form below to make a report.
+
+**REPORTING FORM:** << [*https://URL/to/your/CoC/reporting/form*](https://URL/to/your/CoC/reporting/form) >>
+
+Reports are sent to the {{ cookiecutter.project_name }} Code of Conduct Enforcement Team (see below).
+
+<!-- If you are using TypeForm, leave the following text in place -->
```

Comment:
I think that you could remove  this comment and the paragraph above about the privacy policy ... 
maybe you can just add a note, something like : `if the form you are using has any privacy policy, please add a paragraph about that with the link.`

Reply:
You can view the Privacy Policy and Terms of Service for TypeForm at [*https://terms.typeform.com/to/fJAmVc*](https://terms.typeform.com/to/fJAmVc).